setup-dev-env.sh

```diff
@@ -84,11 +86,15 @@ fi
 # Check installation of dev package
 if [ "$option_no_dev" = "true" ]; then
     ansible_args+=("--extra-vars" "install_devel=false")
-    ansible_args+=("--extra-vars" "installation_type=ros-base")
```

Comment:
How about merging the PR of `--no-dev` to `--runtime` first and including `installation_type=ros-base` to `--runtime`?
https://github.com/autowarefoundation/autoware/pull/2922#discussion_r1119470655

Reply:
The reason is that I think it's not natural to add `option_ros_install_base`.

Reply:
Sure, I'll create a separate PR for this.